src/main/java/com/application/bookingservice/dto/customer/CustomerDto.java

```diff
@@ -0,0 +1,16 @@
+package com.application.bookingservice.dto.customer;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class CustomerDto {
```

Comment:
CustomerRequestDto

Reply:
agree

Reply:
And than make separately dto for  response. Cause you have a method where you update customer and get in param and return this CustomerDto. So you can make CustomerUpdateRequestDto and CustomerUpdateResponseDto. They will have the same params - OK